src/ecdsa/ot_based_ecdsa/test.rs

```diff
@@ -35,6 +51,57 @@ pub fn run_sign(
     )
 }
 
+type SigWithRerand = (Tweak, Vec<(Participant, Signature)>);
```

Comment:
I learn recently that it is better to avoid this
```suggestion
type SignatureWithRerandomization = (Tweak, Vec<(Participant, Signature)>);
```

Reply:
This goes away entirely with #93 I will not modify it not to cause unecessary conflicts